routes/movies.routes.js

```diff
@@ -0,0 +1,62 @@
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+router.get('/create', (req, res) => {
+    Celebrity.find()
+    .then(celebs => {
+        res.render('movies/new-movie',{celebs})
+    })
+})
+
+router.post('/create', (req, res) => {
+    const {title, genre, plot, cast} = req.body
+    Movie.create({title, genre, plot, cast})
+    .then(movie => {
+        console.log(movie)
+        res.redirect('/movies')
+    })
```

Comment:
Don't forget `.catch()` :) 

```suggestion
    })
    .catch(error => /* Do something with the error*/)
```